### PR TITLE
WIP: Mendix-to-Mendix Write 

### PR DESCRIPTION
**Pull request by ezagroba**
Adding details from Integration, Catalog, and Studio Pro to describe Mendix-to-Mendix Write to users.

### CODE REVIEW

**Comment by ezagroba** [comment]
Hold off until all three teams are ready with their features here.

**Review by mendixmiles — COMMENTED** [review]
Line 37 - plublish - should be 'publish'
Suggest rewording sentence on line 45 -'You'll be able to see which capabilities which entities, attributes, and associations have. Actually making the changes to that data happens in your running Mendix app. ' - suggest "You can see the capabilities of entities, attributes and associations respectively. Changes to the source data occurs via the Mendix app that owns the data, not the Data Hub itself."

**Comment by ezagroba** [comment]
@MariaShaposhnikova @ConnorLand Ready for review.

File: content/howto/write-data/index.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+title: "How to Write Data to Another App with Data Hub"
```
Comment (Adam-Dupaski) [inline content/howto/write-data/index.md:2]: @MariaShaposhnikova This should probably go in https://docs.mendix.com/howto/integration/ - no need for a new category just for this how-to.

Comment (MariaShaposhnikova) [reply content/howto/write-data/index.md:2]: @Adam-Dupaski , @ezagroba , I've transferred it to the Data Hub catalog > under Share Data Between Apps. Since this how-to adds information to the Share Data Between Apps, I think it makes sense to keep them in one guide.

**Comment by MariaShaposhnikova** [comment]
@ezagroba , just to double-check, are we sure all data on screenshots can be published? Particularly image **crud-capabilities** in **write-data document** and all images in the **data-hub-pane** document.

File: content/howto/write-data/index.md (hunk 1 above)
Comment (ezagroba) [reply content/howto/write-data/index.md]: Certainly, thank you. I appreciate your guidance on this.

**Comment by ezagroba** [comment]
@MariaShaposhnikova I double-checked all the screenshots and edited out the one that had an employee's name in it. They're all good now.
